app/controllers/kittens_controller.rb

```diff
@@ -7,4 +7,30 @@ def index
 
     render layout: true
   end
+
+  def new
+    # TODO
+    # @kitten = Kitten.new
```

Comment:
Create a class in `app/models/kitten.rb`

```
class Kitten < ActiveRecord::Base
  # Exemplary relation with project
  belongs_to :project

  validates_presence_of :name

  def some_method
    "This is kitten #{name} belonging to #{project.name}"
  end
end
```

Add a migration for the respective table `db/migrate/20170203214512_add_kittens_table.rb`


```
class AddKittensTable < ActiveRecord::Migration[5.0]
  def change
    create_table :kittens do |t|
      t.string :name, null: false
      # Exemplary relation with project
      t.column :project_id, :integer
    end
  end
end
```